themes-default/slim/views/config_subtitles.mako

```diff
@@ -1,7 +1,12 @@
 <%inherit file="/layouts/main.mako"/>
 <%!
+    import json
     from medusa import subtitles
     from medusa import app
+
+    services = []
+    for service in subtitles.sorted_service_list():
```

Comment:
This can be just `services = subtitles.sorted_service_list()`.

Reply:
If I put `subtitles.sorted_service_list()` straight through the json.dumps I get a mako error.

Reply:
@OmgImAlexis What error?